src/main/kotlin/com/atlassian/performance/tools/report/api/judge/RelativeNonparametricPerformanceJudge.kt

```diff
@@ -62,7 +62,7 @@ class RelativeNonparametricPerformanceJudge(
         val test = ShiftedDistributionRegressionTest(baseline, experiment, mwAlpha = significance, ksAlpha = 0.0)
         return if (test.isExperimentRegressed(toleranceRatio.toDouble())) {
             val confidenceLevelPercent = (1.0 - significance).toPercentage(decimalPlaces = 2)
-            val message = "There is a regression with $confidenceLevelPercent confidence level. Regression is larger than allowed ${toleranceRatio.toPercentage(decimalPlaces = 2)} tolerance"
+            val message = "There is a regression in [$label] with $confidenceLevelPercent confidence level. Regression is larger than allowed ${toleranceRatio.toPercentage(decimalPlaces = 2)} tolerance"
```

Comment:
This is a weird thing to maintain. I'd leave it as a problem for the consumer that was relying on it.
OTOH, if the consumer is right, then we need to promote this to a behavioral contract, ie. promise the substring in Javadoc.

Reply:
So we either have an unnecessary change or a new API, in both cases the diff is insufficient (either Javadoc or changelog). We shouldn't release this in this state.

Reply:
You're right. I'm figuring out how to not rely on the log message in first place in JPT test

Reply:
I'll add unit tests for RelativeNonparametricPerformanceJudge in this library, and JPT should use mocked results instead of relying on output from this class.
More work than promising substring in javadoc, but reliable

Reply:
> I'll add unit tests for RelativeNonparametricPerformanceJudge in this library

Beautiful :heart: 

> JPT should use mocked results 

Good idea. Also, iJPT could relax the assertion.

Reply:
I didn't see it and changelog already contains 3.11.4 release. However I like your message more and will use it in next PR with unit tests